runtime/queries/arduino/highlights.scm

```diff
@@ -1,96 +1,24 @@
 ; inherits: cpp
-
 ((identifier) @function.builtin
- (#any-of? @function.builtin
-           ; Digital I/O
-           "digitalRead"
-           "digitalWrite"
-           "pinMode"
-           ; Analog I/O
-           "analogRead"
-           "analogReference"
-           "analogWrite"
-           ; Zero, Due & MKR Family
-           "analogReadResolution"
-           "analogWriteResolution"
-           ; Advanced I/O
-           "noTone"
-           "pulseIn"
-           "pulseInLong"
-           "shiftIn"
-           "shiftOut"
-           "tone"
-           ; Time
-           "delay"
-           "delayMicroseconds"
-           "micros"
-           "millis"
-           ; Math
-           "abs"
-           "constrain"
-           "map"
-           "max"
-           "min"
-           "pow"
-           "sq"
-           "sqrt"
-           ; Trigonometry
-           "cos"
-           "sin"
-           "tan"
-           ; Characters
-           "isAlpha"
-           "isAlphaNumeric"
-           "isAscii"
-           "isControl"
-           "isDigit"
-           "isGraph"
-           "isHexadecimalDigit"
-           "isLowerCase"
-           "isPrintable"
-           "isPunct"
-           "isSpace"
-           "isUpperCase"
-           "isWhitespace"
-           ; Random Numbers
-           "random"
-           "randomSeed"
-           ; Bits and Bytes
-           "bit"
-           "bitClear"
-           "bitRead"
-           "bitSet"
-           "bitWrite"
-           "highByte"
-           "lowByte"
-           ; External Interrupts
-           "attachInterrupt"
-           "detachInterrupt"
-           ; Interrupts
-           "interrupts"
-           "noInterrupts"
-           ))
+  (#any-of? @function.builtin ; Digital I/O
```

Comment:
OK; _that_ is an argument for allowing to ignore queries (or patterns)...

Reply:
If it's feasible, supporting `; format-ignore` for individual patterns would be best!

Reply:
> If it's feasible, supporting `; format-ignore` for individual patterns would be best!

That's easy. Adding a pattern to support that and I think it should be good to go

Reply:
Added the pattern.